update.go

```diff
@@ -363,6 +363,10 @@ other options are ignored.
 				}
 				config.IntelRdt = &configs.IntelRdt{}
 				intelRdtManager := intelrdt.NewManager(&config, container.ID(), state.IntelRdtPath)
+				if intelRdtManager == nil {
+					logrus.Errorf("Intel RDT is not available: resctrl filesystem not found or unsupported")
```

Comment:
I think you misunderstood my comment, you should add the log in here: 
https://github.com/opencontainers/runc/blob/314b79aeca1bad2a850e92b85907564f98ec8060/libcontainer/intelrdt/intelrdt.go#L162-L165

Because no one knows what’s contents in this ‘err’.